### PR TITLE
taskIdList query add

### PR DESCRIPTION
**Pull request by qiudaoke**
#### Check List:
* Unit tests: YES
* Documentation: YES

### CODE REVIEW

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/history/HistoryServiceTaskIdInTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.flowable.engine.runtime.ProcessInstance;
+import org.flowable.engine.test.Deployment;
+import org.flowable.task.api.Task;
+import org.flowable.task.api.history.HistoricTaskInstance;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (dbmalkovsky) [inline modules/flowable-engine/src/test/java/org/flowable/engine/test/api/history/HistoryServiceTaskIdInTest.java:26]: Thanks for the PR.

Please follow project coding standards for imports; see https://github.com/flowable/flowable-engine/tree/main/ide-settings.

The standard order is:

static imports
java
javax
org
com
everything else

Comment (qiudaoke) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/history/HistoryServiceTaskIdInTest.java:26]: ok

**Comment by filiphr** [comment]
Manually merged this in https://github.com/flowable/flowable-engine/commit/6eda43e2af7a058421be056d36bc8849ea8020c7

**Comment by filiphr** [comment]
Thanks @qiudaoke

**Pull request closed without merging**